src/commands/kthid/kthid.command.ts

```diff
@@ -0,0 +1,15 @@
+import { PermissionFlagsBits, SlashCommandBuilder } from "discord.js";
+import { CommandNames } from "../commands.names";
+import { KthIdVariables } from "./kthid.variables";
+
+export const kthIdCommand = new SlashCommandBuilder()
+	.setName(CommandNames.KTHID)
+	.setDescription("Get the KTH ID of a user")
+	.setDefaultMemberPermissions(PermissionFlagsBits.ManageGuild);
+
+kthIdCommand.addStringOption((option) =>
```

Comment:
Kan vara bra att ha UserOption istället så att man får Discords inbyggda felmeddelande för ifall man matar in något annat än en user. Också lite bättre att inte behöva plocka ut användar-ID med regex.
```suggestion
kthIdCommand.addUserOption((option) =>
```